src/ostree/ot-builtin-find-remotes.c

```diff
@@ -232,14 +232,14 @@ ostree_builtin_find_remotes (int            argc,
   /* Build the array of OstreeRepoFinder instances */
   if (opt_finders != NULL)
     {
-      g_auto(GStrv) finders_strings = NULL;
+      g_autofree const char **finders_strings = NULL;
 
-      finders_strings = g_strsplit (opt_finders, ",", 0);
+      finders_strings = (const char **)g_strsplit (opt_finders, ",", 0);
       if (!validate_finders_list (finders_strings, context, error))
         return FALSE;
 
-      finders = g_ptr_array_new ();
-      for (char **iter =finders_strings; iter && *iter; iter++)
+      finders = g_ptr_array_new_with_free_func (NULL);
```

Comment:
Any reason for this? Did you mean to use `g_object_unref` as the free function here and get rid of the local `finder_*` vars?

Reply:
@pwithnall suggested it as a way to make it clear that there's intentionally not a `free_func`